app/assets/javascripts/admin/addon/routes/admin-user-index.js

```diff
@@ -10,19 +11,20 @@ export default class AdminUserIndexRoute extends DiscourseRoute {
     return this.currentModel.username;
   }
 
-  afterModel(model) {
+  async afterModel(model) {
     if (this.currentUser.admin) {
-      return Group.findAll().then((groups) => {
-        this._availableGroups = groups.filterBy("automatic", false);
-        return model;
-      });
+      const groups = await Group.findAll();
+      this._availableGroups = groups.filterBy("automatic", false);
+
+      this._userExport = await UserExport.findLatest(model.id);
```

Comment:
Could we include this object in the admin user model object instead of trigger another request?

Reply:
I agree this would make more sense, it's an addition to the serializer then you can convert it to a `UserExport` JS model using `UserExport.create(obj)`

Reply:
Good idea, got that sorted.